x-pack/plugins/infra/public/pages/metrics/inventory_view/components/waffle/node_square.tsx

```diff
@@ -0,0 +1,165 @@
+/*
```

Comment:
The content has not changed: I just moved it to a new component so we have a clear separation and better readability

Reply:
I was thinking about that but didn't want to change the old component content to avoid confusion. 

One note regarding the structure there: I saw that the styles are a bit more complex than most places so for some reason when I tried to use a similar composition as in the example `base`in the [docs](https://emotion.sh/docs/composition) but for some reason I got this error: 
```
You have tried to stringify object returned from `css` function. It isn't supposed to be used directly (e.g. as value of the `className` prop), but rather handed to emotion so it can handle it (e.g. as value of `css` prop).
```
So I ended up defining the styles in an object and using them as 
```
css={css`${styles....}`}`
```